io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md

```diff
@@ -114,6 +114,19 @@ issuer=https://localhost:9443/oauth2/token
 jwksEndpoint=https://localhost:9443/oauth2/jwks
 ```
 
+
+### Logout URL
+
+**Property Name:** `logoutURL`
+
+**Description:** This is the URL to which the user will be directed upon a logout request.
```

Comment:
logoutURL is the URL to denote the logout endpoint for the application. Let's reword the description to capture that.

Reply:
Resolved via [bf5e232](https://github.com/asgardeo/asgardeo-tomcat-oidc-agent/pull/42/commits/bf5e2323c61ba6b2673ac881213eefe1a29e579a)